Readme.md

```diff
@@ -146,6 +146,39 @@ Start your file with a module docstring. Do not put a shebang line (`#!/usr/bin/
 
 > *Rationale:* a shebang line is useless for non-executable files. An `__author__` line just gets out of date, and is better determined by looking at source control history in any case. Code is automatically copyrighted; a copyright line doesn't help. No need to put this useless boilerplate at the top of the file!
 
+## Python Tests
```

Comment:
"Python" is redundant here because its the Python style guide :) just "Tests" will do